src/client.js

```diff
@@ -140,7 +140,7 @@ Client.prototype.find = function(channelId) {
 
 Client.prototype.connect = function(args) {
 	const client = this;
-	const nick = args.nick || "lounge-user";
+	const nick = args.nick || "thelounge";
```

Comment:
This was changed in https://github.com/thelounge/thelounge/commit/84e524c6352e3a7f6c093585d408c0582568760d. However, I'm wondering if we actually still need this fallback.

Reply:
We do need a fallback, I've changed it in #2229 by the way.

Reply:
Mind reminding me what this fallback is for, vs the one in config defaults?

Reply:
Actually good question as to why we don't use the one from `defaults`. But the fallback is to disallow connecting with an empty nick.

I'll fix the default in my PR.

Reply:
Cool! Feel free to remove from my pr if you're happy with the rest and ok to merge